pybliometrics/scopus/tests/test_AbstractRetrieval.py

```diff
@@ -225,6 +227,9 @@ def test_endingPage():
     assert_equal(ab1.endingPage, '3899')
     assert_equal(ab8.endingPage, None)
 
+def test_entitlement():
+    entitlement = ar_entiteled.document_entitlement_status
+    assert_equal(entitlement, 'ENTITLED')
```

Comment:
Also test how some of the other objects behave (they should all behave in same way, right?).

Reply:
Every other property is none. The entitlement tells the user if it is entitled to access the resource.

Reply:
Yes I know that. I want you to include this as a test case, like in https://github.com/pybliometrics-dev/pybliometrics/blob/master/pybliometrics/scopus/tests/test_AuthorRetrieval.py#L105.